### PR TITLE
Separate okapi compile methods for node and browser

### PR DESCRIPTION
**Pull request by tmarkovski**

### CODE REVIEW

**Review by MichaelEdwardBlack — APPROVED** [review]

File: wasm/examples/node-sample/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "author": "Tomislav Markovski",
   "license": "ISC",
   "dependencies": {
-    "@trinsic/okapi": "1.0.0-preview.19"
+    "@trinsic/okapi": "file:../../packages/okapi"
```
Comment (fundthmcalculus) [inline wasm/examples/node-sample/package.json:13]: Is this guaranteed to be here?

File: wasm/examples/webpack-sample/package.json [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
   "scripts": {
     "build": "webpack build",
+    "start": "npx http-server",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Tomislav Markovski",
   "license": "ISC",
   "dependencies": {
-    "@trinsic/okapi": "1.0.0-preview.19"
+    "@trinsic/okapi": "file:../../packages/okapi"
```
Comment (fundthmcalculus) [inline wasm/examples/webpack-sample/package.json:14]: Guaranteed path?

**Review by fundthmcalculus — APPROVED** [review]

File: wasm/examples/node-sample/package.json (hunk 1 above)
Comment (tmarkovski) [reply wasm/examples/node-sample/package.json:13]: Only after a build, yes. The idea is for this sample to actually use the local version of the package, instead npm one.
This is why I made the `okapi-examples` repo which will use these packages as intended in production.

File: wasm/examples/webpack-sample/package.json (hunk 2 above)
Comment (tmarkovski) [reply wasm/examples/webpack-sample/package.json:14]: Yes, see other comment.

**Pull request merged**